Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/update/DccWalletInfoUpdateTrigger.kt

```diff
@@ -10,11 +10,12 @@ class DccWalletInfoUpdateTrigger @Inject constructor(
     private val taskController: TaskController
 ) {
 
-    fun triggerDccWalletInfoUpdate() {
+    fun triggerDccWalletInfoUpdate(configurationChanged: Boolean = false) {
```

Comment:
I think we need to make `true` by default? we trigger calculation update under different circumstances:
1- Register Dcc
2- Recycle Dcc
3- Restore Dcc
4- Retreive Dcc

upcoming sprint 
5- Replace Dcc 

it would be nicer, to just call  `triggerDccWalletInfoUpdate()` without worrying about his flag 
on the other hand this flag is only provides for CCL configuration update where we would know if there is update or not , so 5 : 1 most cases will require an update immediately

Reply:
I think we should have 2 differert triggers:
- One that is called after ccl / booster updates (1)
- One that is called after changes in certificates (2)

(1) can continue to call `DccWalletInfoCalculationManager.triggerCalculation()`. Here we have some checks in which cases we do a recalculation (If either the configuration got updated or a dccWalletInfo got invalid). Sometimes, no calculation takes place. 

(2) should call a new function in `DccWalletInfoCalcluationManager`, since in this case we don't need to have any checks and a calculation always needs to be performed.

What do you think?

Reply:
Makes sense, please go ahead?

Reply:
done